src/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/ExternalLoginServiceTests.cs

```diff
@@ -202,6 +203,31 @@ public void Add_Logins()
             }
         }
 
+        [Test]
+        public void Add_Tokens()
+        {
+            var user = new User(GlobalSettings, "Test", "test@test.com", "test", "helloworldtest");
```

Comment:
I know it is just copy/pasted, but when we don't use any of this dummy parameters, we really should use our builder. This suggestion can be done for the entire test class

```suggestion
            var user = new UserBuilder().Build();
```